R/drive_mkdir.R

```diff
@@ -54,9 +55,18 @@ drive_mkdir <- function(path = NULL, name = NULL, verbose = TRUE) {
 
   path <- path %||% root_folder()
   path <- as_dribble(path)
-  confirm_single_file(path)
+  if (!single_file(path)) {
```

Comment:
Ah ok. This is probably good time to do yet another consistency check across `drive_cp()`, `drive_mv()`, `drive_mkdir()`. Would it simplify some of those other functions to do same, i.e. default to parent being root folder?